shard.go

```diff
@@ -234,7 +234,9 @@ func (s *Shard) close() error {
 	if s.store != nil {
 		_ = s.store.Close()
 	}
-	s.stats.Inc("close")
```

Comment:
https://github.com/influxdb/influxdb/blob/master/server.go#L275 is where shards can be closed that were never opened.